File: python/prism_sdk/local_knowledge_representation_inference.py
"""Worldgen P04 AFA-worldgen-P04-F01 product wrapper."""
from .worldgen_knowledge_representation_support import KnowledgeRepresentationRequest, KnowledgeRepresentationReceipt, manifest as _manifest, represent as _run
FEATURE_ID="AFA-worldgen-P04-F01"
CONTRACT_VERSION="worldgen-local-knowledge-representation/1.0"
def worldgen_local_knowledge_representation_inference_manifest(): return _manifest(feature_id=FEATURE_ID, contract_version=CONTRACT_VERSION, input_schema="KnowledgeRepresentationRequest1@1", scale="local single-study", autonomy_tier="A0")
def represent_worldgen_local_knowledge(request): return _run(request, feature_id=FEATURE_ID, contract_version=CONTRACT_VERSION, scale="local single-study", require_federation=False)
__all__=["KnowledgeRepresentationRequest","KnowledgeRepresentationReceipt","worldgen_local_knowledge_representation_inference_manifest","represent_worldgen_local_knowledge"]

File: crates/worldgen/src/local_knowledge_representation_inference.rs
//! Worldgen P04-F01 local knowledge-representation inference engine.
use super::knowledge_representation_support::{self, KnowledgeRepresentationRequest, KnowledgeRepresentationReceipt};
pub const FEATURE_ID:&str="AFA-worldgen-P04-F01"; pub const CONTRACT_VERSION:&str="worldgen-local-knowledge-representation/1.0";
pub fn worldgen_local_knowledge_representation_inference_manifest()->serde_json::Value{knowledge_representation_support::manifest(FEATURE_ID,CONTRACT_VERSION,"KnowledgeRepresentationRequest1@1","local single-study","A0")}
pub fn represent_worldgen_local_knowledge(request:&KnowledgeRepresentationRequest)->Result<KnowledgeRepresentationReceipt,knowledge_representation_support::KnowledgeRepresentationError>{knowledge_representation_support::represent(request,FEATURE_ID,CONTRACT_VERSION,"local single-study",false)}
pub use knowledge_representation_support::{KnowledgeRepresentationError,KnowledgeRepresentationReceipt as WorldgenLocalKnowledgeRepresentationReceipt,KnowledgeRepresentationRequest as WorldgenLocalKnowledgeRepresentationRequest,KnowledgeNode as WorldgenKnowledgeNode,KnowledgeRelation as WorldgenKnowledgeRelation};
